scalardb-graphql-sample/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.9"
+services:
+  cassandra:
+    image: cassandra:3.11
+    container_name: "cassandradb-1"
+    volumes:
+      - cassandra-data:/var/lib/cassandra
+    ports:
+      - "9042:9042"
+    networks:
+      - scalar-network
+
+  schema-loader:
+    image: ghcr.io/scalar-labs/scalardb-schema-loader:3.5.0
+    container_name: "scalardb-schema-loader-1"
+    volumes:
+      - ./database.properties:/database.properties
+      - ./emoney-schema.json:/emoney-schema.json
+    command: "--config /database.properties -f /emoney-schema.json --coordinator"
+    networks:
+      - scalar-network
+    profiles:
+      - tools
+
+  scalardb-graphql:
+    image: ghcr.io/scalar-labs/scalardb-graphql:3.5.0
```

Comment:
Since this image has not been pushed to our registry, it is needed to build with `./gradlew docker`.